menpo/shape/mesh/base.py

```diff
@@ -6,6 +6,8 @@
 from .normals import compute_face_normals, compute_vertex_normals
 from .. import PointCloud
 from ..adjacency import mask_adjacency_array, reindex_adjacency_array
+from scipy.spatial import cKDTree
```

Comment:
These are expensive imports so we should probably inline them

Reply:
Ok, I was trying to comply with PEP8 but we can inline them .